dora/grid.py

```diff
@@ -91,24 +90,31 @@ class RunGridArgs:
 
 def _get_explore(args, main):
     # Finds the explorer.
-    root_name = main.package + ".grids"
-    grids = import_or_fatal(root_name)
+    grid_package = main.dora.grid_package
```

Comment:
Instead of defining an additional variable `main.dora.grid_package`, would it be possible to just check if `Path(args.grid).exists()` and if it does use this path for the grid file instead?

This way you can use any file as a grid by just running `dora grid foo/bar.py`

Reply:
nope not possible, as I said in the chat: the grid name is used in a folder to where symlinks are stored to all experiments inside the grid. This folder name would become non reliable if a path would be used (because different paths could point to the same file) and more improtantly it would become impossible to use this path as a filename.

Reply:
A solution to that would be to specify the unique grid name as input to the dora.Explorer decorator for instance. Just mentioning in case we might want to reconsider in the future. For now it's fine.